Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -699,8 +728,8 @@ theorem nnnorm_mul_le' (a b : E) : ‖a * b‖₊ ≤ ‖a‖₊ + ‖b‖₊ :=
   NNReal.coe_le_coe.1 <| norm_mul_le' a b
 
 @[to_additive enorm_add_le]
-lemma enorm_mul_le' (a b : E) : ‖a * b‖ₑ ≤ ‖a‖ₑ + ‖b‖ₑ := by
-  simpa [enorm, ← ENNReal.coe_add] using nnnorm_mul_le' a b
+lemma enorm_mul_le' {E : Type*} [TopologicalSpace E] [ENormedMonoid E] (a b : E) :
+  ‖a * b‖ₑ ≤ ‖a‖ₑ + ‖b‖ₑ := ENormedMonoid.enorm_mul_le a b
```

Comment:
Can this become an earlier section?

Reply:
I have moved this result to an earlier section. The other results (the instances or the continuity statement) cannot, since they require material from a section further down).

Reply:
Did you forget to commit? This result hasn't moved since I commented on it

Reply:
I could've sworn I made this change, but I don't see it locally either. I just moved it.